src/main/java/com/dislinkt/post/controller/ReactionController.java

```diff
@@ -0,0 +1,94 @@
+package com.dislinkt.post.controller;
+
+import java.util.List;
+import java.util.UUID;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.dislinkt.post.dto.ErrorDTO;
+import com.dislinkt.post.dto.ReactionDTO;
+import com.dislinkt.post.dto.ResponseDTO;
+import com.dislinkt.post.service.ReactionService;
+
+@RestController
+@CrossOrigin
+@RequestMapping("/reactions")
+public class ReactionController {
+
+    @Autowired
+    private ReactionService service;
+
+    @GetMapping()
+    public ResponseEntity<ResponseDTO<List<ReactionDTO>>> getAll(){
+        List<ReactionDTO> ret = service.findAll();
+        return new ResponseEntity<>(new ResponseDTO<>(ret), HttpStatus.OK);
+    }
```

Comment:
Do we need this endpoint at all? We just want to query reactions by each post I guess

Reply:
We don't need it in general, but it is useful for checks in Postman (at least for now while we have no frontend)